monitoring/prometheus/alertmanager.yml

```diff
@@ -0,0 +1,38 @@
+global:
+  resolve_timeout: 10s
+  slack_api_url: 'https://hooks.slack.com/services/x/x/x'
+
+route:
+  receiver: 'slack-default-notifications'
+  group_interval: 10s
+  repeat_interval: 10s
+  routes:
+  - matchers: [group = api]
+    receiver: 'slack-apis-notifications'
+  - matchers: [group = instance]
+    receiver: 'slack-instance-notifications'
+
+receivers:
+  - name: 'slack-default-notifications'
+    slack_configs:
+      - channel: '#x'
+        title: '{{ .commonAnnotations.title }}'
+        text: '{{ .commonAnnotations.description }}'
+        send_resolved: false
+  - name: 'slack-apis-notifications'
+    slack_configs:
+      - channel: '#x'
+        title: '{{ template "slack.apis.title" . }}'
+        text: '{{ template "slack.apis.text" . }}'
+        send_resolved: false
```

Comment:
Why do we have multiple receivers? There should be one receiver channel, no?

Reply:
@Ram81 I have grouped all the possible alerts into two groups - Instance Alerts and API alerts. Both of these different templates due to different tags they have. For each alert group, we need to define receivers cause even though the channel is the same, we also provide the template configs here. Though, we can remove the default-receiver which I had kept to make the code a bit generic and use that receiver for any alerts that do not group under the api or instance groups. However, at the moment we are not emitting any such alerts. So, I added this as it may be needed sometime in the future

Reply:
@burnerlee please remove the default config. We only need to add the ones we are using